src/logging.c

```diff
@@ -24,81 +24,271 @@ extern "C"
 #include "rcutils/get_env.h"
 #include "rcutils/logging.h"
 #include "rcutils/snprintf.h"
+#include "rcutils/types/string_map.h"
 
 #define RCUTILS_LOGGING_MAX_OUTPUT_FORMAT_LEN 2048
 
+const char * g_rcutils_log_severity_names[] = {
+  [RCUTILS_LOG_SEVERITY_UNSET] = "UNSET",
+  [RCUTILS_LOG_SEVERITY_DEBUG] = "DEBUG",
+  [RCUTILS_LOG_SEVERITY_INFO] = "INFO",
+  [RCUTILS_LOG_SEVERITY_WARN] = "WARN",
+  [RCUTILS_LOG_SEVERITY_ERROR] = "ERROR",
+  [RCUTILS_LOG_SEVERITY_FATAL] = "FATAL",
+};
```

Comment:
How are the other entries initialized?

Reply:
automatically as null, there's a check for null after retrieving the values at: https://github.com/ros2/rcutils/pull/57/files/ab249daaef9556c9aa2a2b79bc0ccb4a98829b61#diff-f909d0b1a298531f76132943dc55f86aR190